ci/README.md

```diff
@@ -0,0 +1,10 @@
+# Theia CI
+
+Project Theia follows [Antrea CI].
+
+## Notice
+
+1. At the moment, we don't support integration test in Theia.
+2. For information about our Jenkins CI jobs, please see [here](jenkins/README.md)
+
+[Antrea CI]: https://github.com/antrea-io/antrea/blob/main/ci/README.md
```

Comment:
Probably we should change this one to be inline.

Reply:
Fixed. Thank you.